data-es6.js

```diff
@@ -18316,7 +18319,7 @@ exports.tests = [
         var desc = Object.getOwnPropertyDescriptor(Object.prototype,"__proto__");
         var A = function(){};
 
-        return (desc
+        return (typeof desc === "object"
```

Comment:
```suggestion
        return (!!desc
```
I think the author's concern is that a test should always return a boolean value. In that case we can cast `desc` to boolean so the test returns `false` instead of `undefined`.

Reply:
returning false and undefined are treated the same. Why does it need to return a boolean?